day3/README.md

```diff
@@ -44,13 +44,97 @@ React.render(<Game data={ data } />, document.body);
 
 In this extract, both Score and Game are view components that we used last time. The Game component are the one being rendered by React, and this component is using the Score component. In the render function, we are passing the data object as a prop to the Game component, which are passed on down to the Score component. At last the Score component are using the receiving prop to show the score in a div.
 
-This was just an example of one child component, but there are now problem with React to create as many children as you want. 
+This was just an example of one child component, but there are now problem with React to create as many children as you want.
 
-### Redux
-store, reducer, action (objekt med evt type), dispatch, rerendring billig med React, devtools!, objkt assign
+## Redux
+All applications needs a way to handle state (server response, cached data, selected button or filter etc.). And as an app grows larger and becomes more complex, a simple way to handle the state is essential.
 
+Redux is a predictable state container which means that, by imposing restriction on how and when updates can happen, you can be sure that you always know what your state will be like.
 
-# Unidirectional data flow
+So let´s dig deeper into the most important concepts of Redux.
+
+//store, reducer, action (objekt med evt type), dispatch, rerendring billig med React, devtools!, objkt assign
+
+### Actions
+Actions are the way to pass data in Redux. It's just plain JavaScript objects that consist of type, and if you want, you can also pass data payload with the action.
+
+Example
+```javascript
+const GAME_RESET = 'GAME_RESET'
+{
+  type: GAME_RESET
+}
+```
+
+#### Action creator
+Action creators are functions that create actions. To trigger an action, you'll need to run the dispatch function with the action.
+
+Example
+```javascript
+function resetGame(index) {
+  return {
+    type: GAME_RESET
+  }
+}
+
+dispatch(resetGame())
+```
```

Comment:
Ikke annet enn i setningen over "..To trigger an action, you'll need to run the dispatch function with the action." Burde vi forklare dispatch nærmere?